crates/ruff_python_formatter/src/trivia.rs

```diff
@@ -185,6 +185,10 @@ pub(crate) enum TokenKind {
     /// `.`.
     Dot,
 
+    /// `:=`.
+    /// TODO: name?
```

Comment:
[walrus is correct](https://docs.python.org/3/whatsnew/3.8.html#assignment-expressions)